locales/pt/json.json

```diff
@@ -757,9 +754,8 @@
     "Turks and Caicos Islands": "Ilhas Turcas e Caicos",
     "Tuvalu": "Tuvalu",
     "Two Factor Authentication": "Autenticação de dois fatores",
-    "Two factor authentication is now enabled. Scan the following QR code using your phone's authenticator application or enter the setup key.": "A autenticação de dois factores está agora ativa. Digitalize o seguinte código QR utilizando a aplicação autenticadora do seu telemóvel ou introduza a chave de configuração.",
+    "Two factor authentication is now enabled. Scan the following QR code using your phone's authenticator application or enter the setup key.": "A autenticação de dois fatores está agora ativa. Digitalize o seguinte código QR utilizando a aplicação autenticadora do seu telemóvel ou introduza a chave de configuração.",
     "Type": "Tipo",
-    "Uganda": "Uganda",
```

Comment:
Keys do not need to be deleted. When translations match, they are simply specified in the "excludes" file.